tests/js/spec/components/events/interfaces/utils.spec.jsx

```diff
@@ -76,6 +77,40 @@ describe('components/interfaces/utils', function() {
         })
       ).toEqual('curl \\\n "http://example.com/foo?foo=bar"');
     });
+
+    it('works with a Proxy', function() {
+      const spy = jest.spyOn(MetaProxy.prototype, 'get');
+      const data = {
+        fragment: '',
+        cookies: [],
+        inferredContentType: null,
+        env: {
+          SERVER_NAME: 'sentry',
+          SERVER_PORT: '443',
+          REMOTE_ADDR: '127.0.0.1',
+        },
+        headers: [
+          ['Accept-Language', 'en'],
+          ['Referer', 'http://example.com'],
+          [
+            'User-Agent',
+            'Mozilla/5.0 (Windows NT 6.2; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/28.0.1500.72 Safari/537.36',
+          ],
+          ['Content-Type', 'application/json'],
+          ['Referer', 'http://example.com'],
+          ['Accept-Encoding', 'gzip'],
+        ],
+        url: 'https://www.sentry.io',
+        query: '',
+        data: null,
+        method: 'GET',
+      };
+      const eventWithProxy = withMeta(data);
+      getCurlCommand(eventWithProxy);
+
+      // This may need to change, but we should aim to keep this low
+      expect(spy).toHaveBeenCalledTimes(172);
```

Comment:
Should this scale with the number of properties in the proxied object graph?

Reply:
It does, but not linearly - I just wanted to capture a baseline for the number of times this is called (before optimization it was about 6-10x more)